interfaces/cython/cantera/ck2yaml.py

```diff
@@ -59,6 +57,21 @@
 except ImportError:
     from ruamel import yaml
 
+# yaml.version_info is a tuple with the three parts of the version
+yaml_version = yaml.version_info
+# We choose ruamel.yaml 0.15.34 as the minimum version
+# since it is the highest version available in the Ubuntu
+# 18.04 repositories and seems to work. Older versions such as
+# 0.13.14 on CentOS7 and 0.10.23 on Ubuntu 16.04 raise an exception
+# that they are missing the RoundTripRepresenter
+yaml_min_version = (0, 15, 34)
+if yaml_version < yaml_min_version:
+    raise RuntimeError(
+        "The minimum version of ruamel.yaml is 0.15.34. If you installed "
```

Comment:
```suggestion
        "The minimum supported version of ruamel.yaml is 0.15.34. If you installed "
```
And ditto in the other scripts.